examples/demo/wicket/jdo/src/main/java/demoapp/webapp/wicket/jdo/DemoAppWicketJdo.java

```diff
@@ -78,10 +78,15 @@ public static void main(final String[] args) {
     	IsisPresets.prototyping();
         //IsisPresets.logging(WebRequestCycleForIsis.class, "debug");
 
-        System.setProperty("spring.profiles.active", "demo-jdo");
+        System.setProperty("spring.profiles.active", preservingAnyExisting("demo-jdo"));
```

Comment:
logic for adding a spring profile, could be refactored into a common (shared) utility;
eg. package `org.apache.isis.core.config.util`

Reply:
rebased on master and picked up improved handling of this.